packages/common-lib/src/index.tsx

```diff
@@ -23,7 +23,12 @@ import * as questionRegistryService from './services/questionRegistryService'
 import * as likeRegistryService from './services/likeRegistryService'
 import * as commentRegistryService from './services/commentRegistryService'
 import * as assessmentRegistryService from './services/assessmentRegistryService'
+<<<<<<< HEAD
```

Comment:
Hey @Hardik-hi the conflicts has not been resolved properly

Reply:
Hi @Shruti3004 have made the changes, please see